Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -276,10 +323,18 @@ void GameList::ExportWiiSave()
   if (export_dir.isEmpty())
     return;
 
-  if (WiiSave::Export(GetSelectedGame()->GetTitleID(), export_dir.toStdString()))
-    QMessageBox::information(this, tr("Save Export"), tr("Successfully exported save files"));
-  else
-    QMessageBox::critical(this, tr("Save Export"), tr("Failed to export save files."));
+  auto games = GetSelectedGames();
+  for (int i = 0; i < games.size(); i++)
+  {
+    auto game = games[i];
+    if (!WiiSave::Export(game->GetTitleID(), export_dir.toStdString()))
+    {
+      QMessageBox::critical(this, tr("Save Export"), tr("Failed to export save files."));
+      return;
```

Comment:
Perhaps it would be nicer to collect the failed files and present them as a list in the end, rather than raising one message box for every failed item (well, actually; only for the first one, and then skipping the rest, which could be a nuisance on its own).

Reply:
Sure, what should I put in the list and how should it be formatted?

Reply:
Thats a very good question. At least some indication of the game(s) that failed (like, the title ID; or better, the name)...not sure if adding the paths/folders is necessary though.

Reply:
Yeah I think I'll just do the name, that should be clear enough, no need to include the ID as well.